### DISCUSSION TITLE
Is there a way to disable a button based on given feed back state?  Say disable buttons if the underlining connection needed has failed?

### DISCUSSION THREAD

**Discussion by Jibun-no-Kage** [bitfocus/companion#2936] [General]
Is there a way to disable a button based on given feed back state?  Say disable buttons if the underlining connection needed has failed?  If connection is OK enable button, if the connection is ERROR disable button, type of thing?

I do have the background of the button(s) change of the connection state if OK or ERROR, but disable of the button would be much better, communicate to the operator button is not going to work.

**Comment by thedist** [comment]
No, it's not possible to disable buttons. Usually you would set up your buttons in a way that you either don't press them if a connection isn't active, or you use actions that don't do anything if the connection isn't active.

**Comment by Julusian** [comment]
There is an open feature request for this, but there are unanswered questions on how it should behave, so nothing has been done on it https://github.com/bitfocus/companion/issues/908

**Comment by Jibun-no-Kage** [comment]
@Julusian, Ah thanks for the reference and update.
